cookbook/form/create_form_type_extension.rst

```diff
@@ -319,3 +311,15 @@ next to the file field. For example::
 
 When displaying the form, if the underlying model has already been associated
 with an image, you will see it displayed next to the file input.
+
+Generic Form Type Extensions
+----------------------------
```

Comment:
I liked the "Generic" wording because it's very easy to search and provides a nice keyword. Plus it matches the text in the bullet points at the top of the article.